trafgen_l3.c

```diff
@@ -81,6 +80,7 @@ static void ipv4_packet_finish(struct proto_hdr *hdr)
 
 	total_len = pkt->len - hdr->pkt_offset;
 	proto_field_set_default_be16(hdr, IP4_LEN, total_len);
+	proto_field_set_default_dev_ipv4(hdr, IP4_SADDR);
```

Comment:
I am thinking that may be it is better to handle failing of getting Ipv4/6 address from device and only warning about this
w/o panic.